src/libopenrave/kinbodystatesaver.cpp

```diff
@@ -18,6 +18,120 @@
 
 namespace OpenRAVE {
 
+/// \brief push link to _listNonCollidingLinksWhenGrabbed of grabbed.
+static void _PushLinkToListNonCollidingLinksWhenGrabbed(Grabbed& grabbed,
```

Comment:
@snozawa comment for readability: I had to check usage of `pLinkRef`. but `pLinkRef->GetIndex()` is guaranteed to be the same as `linkindex` with the current usage. It would be easier to read if you pass only the name of the link to this function.

Reply:
thanks for your comment!

fixed it!

https://github.com/rdiankov/openrave/pull/1429/commits/4b85188527c5b9cbabeaec07349476939b18c6eb